tmail-backend/tmail-third-party/openpaas/src/test/java/com/linagora/tmail/OpenPaasAmqpForwardAttributeIntegrationTest.java

```diff
@@ -134,26 +134,25 @@ void setUpJamesServer(@TempDir File temporaryFolder, String exchange) throws Exc
 
             jamesServer = TemporaryJamesServer.builder()
                 .withBase(Modules.combine(MemoryJamesServerMain.SMTP_AND_IMAP_MODULE,
-                    new OpenPaasModule()))
-                .withOverrides(new AbstractModule() {
-                    @Provides
-                    @Singleton
-                    public RabbitMQConfiguration provideCommonRabbitMQConfiguration() throws URISyntaxException {
-                        return rabbitMQExtension.getRabbitMQ().getConfiguration();
-                    }
-
-                    @Provides
-                    @Singleton
-                    public OpenPaasConfiguration provideOpenPaasConfiguration()
-                        throws URISyntaxException {
-                        return new OpenPaasConfiguration(
-                            AmqpUri.from(rabbitMQExtension.getRabbitMQ().amqpUri()),
-                            URI.create("http://localhost:8081"),
-                            "user",
-                            "password",
-                            false);
-                    }
-                })
+                    new OpenPaasModule(), new AbstractModule() {
+                        @Provides
+                        @Singleton
+                        public RabbitMQConfiguration provideCommonRabbitMQConfiguration() throws URISyntaxException {
+                            return rabbitMQExtension.getRabbitMQ().getConfiguration();
+                        }
+
+                        @Provides
+                        @Singleton
+                        public OpenPaasConfiguration provideOpenPaasConfiguration()
```

Comment:
recipientsShouldBePublishedToAmqpWhenSendingEmailWithExchangeTypeFanoutConfiguration(File) – com.linagora.tmail.OpenPaasAmqpForwardAttributeIntegrationTest$WithOpenPaasModuleAndFanoutRabbitMqExchange
<1s
Error
Unable to create injector, see the following errors:
1) [Guice/BindingAlreadySet]: OpenPaasConfiguration was bound multiple times.